README.md

```diff
@@ -113,6 +115,24 @@ You can use [verify-on-etherscan](https://www.npmjs.com/package/verify-on-ethers
 API_KEY=your-etherscan-api-key yarn verify
 ```
 
+## Verify contract on xDAI
+
+Flatten the smart contract:
+```bash
+npx truffle-flattener contracts/4/BridgedTokenOWL.sol > build/BridgedTokenOWL-BlockScout.sol
+```
+
+Go to Blockscout validation page:
```

Comment:
Would be nice to have the validation also on [sourcify](https://verification.komputing.org/)

Reply:
As far as I can tell sourcify doesn't yet support the xDAI chain.

Reply:
ohhh true ... my fault ... sorry 😬